pybamm/parameters/parameter_values.py

```diff
@@ -265,64 +278,22 @@ def update(self, values, check_conflict=False, check_already_exists=True, path="
                     values[name] = float(value)
             else:
                 self._dict_items[name] = value
-        # check parameter values
-        self.check_and_update_parameter_values(values)
         # reset processed symbols
         self._processed_symbols = {}
 
-    def check_and_update_parameter_values(self, values):
+    def check_parameter_values(self, values):
         # Make sure typical current is non-zero
         if "Typical current [A]" in values and values["Typical current [A]"] == 0:
             raise ValueError(
                 "'Typical current [A]' cannot be zero. A possible alternative is to "
                 "set 'Current function [A]' to `0` instead."
             )
-        if "C-rate" in values and "Current function [A]" in values:
+        if "C-rate" in values:
```

Comment:
are these supposed to be in this PR, doesn't seem like these are anything to do with sensitivities?